File: src/test/java/com/everyonewaiter/EveryonewaiterApiApplicationTests.java
package com.everyonewaiter;

import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.context.annotation.Import;
import org.springframework.test.context.ActiveProfiles;

@ActiveProfiles("test")
@SpringBootTest
@Import(TestcontainersConfiguration.class)
class EveryonewaiterApiApplicationTests {

  @Test
  void contextLoads() {
    // Spring boot application context loads successfully
  }

}

File: src/test/kotlin/com/everyonewaiter/EveryonewaiterApiApplicationTests.kt
package com.everyonewaiter

import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.context.annotation.Import

@Import(TestcontainersConfiguration::class)
@SpringBootTest
class EveryonewaiterApiApplicationTests {

	@Test
	fun contextLoads() {
	}

}
